scipy/optimize/_zeros_py.py

```diff
@@ -1392,3 +1392,125 @@ def toms748(f, a, b, args=(), k=1,
                           maxiter=maxiter, disp=disp)
     x, function_calls, iterations, flag = result
     return _results_select(full_output, (x, function_calls, iterations, flag))
+
+
+def _bracketer(f, a, b=None, min=None, max=None, factor=2, batch=100):
```

Comment:
I would already use keyword only in case this becomes public.

Reply:
Done.